external/source/HostingCLR_inject/HostingCLR/HostingCLR.cpp

```diff
@@ -19,12 +19,7 @@
 #define MethodJittingStarted 145
 #define ILStubGenerated 88
 
-bool amsiflag;
-bool etwflag;
-unsigned char signflag[1];
-
-char sig_40[] = { 0x76,0x34,0x2E,0x30,0x2E,0x33,0x30,0x33,0x31,0x39 };
-char sig_20[] = { 0x76,0x32,0x2E,0x30,0x2E,0x35,0x30,0x37,0x32,0x37 };
+#define ReportErrorThroughPipe(pipe, format, ...) {char buf[1024]; DWORD written; sprintf(buf, format, __VA_ARGS__); WriteFile(pipe, buf, strlen(buf), &written, NULL);}
```

Comment:
I think this needs a size check on `buf` right? 

```suggestion
#define ReportErrorThroughPipe(pipe, format, ...) {char buf[1024]; DWORD written; snprintf(buf, sizeof(buf), format, __VA_ARGS__); WriteFile(pipe, buf, strlen(buf), &written, NULL);}
```

Also it'd be really handy if there could be a standard prefix for all the messages generated by HostingCLR.cpp so we can tell the difference.

Reply:
Done - prefixed with [CLRHOST]